packages/dataviews/src/filter-summary.js

```diff
@@ -6,20 +6,72 @@ import {
 	privateApis as componentsPrivateApis,
 	Icon,
 } from '@wordpress/components';
-import { chevronDown } from '@wordpress/icons';
+import { chevronDown, chevronRightSmall, check } from '@wordpress/icons';
 import { __, sprintf } from '@wordpress/i18n';
+import { Children, Fragment } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
-import { OPERATOR_IN } from './constants';
+import { OPERATOR_IN, OPERATOR_NOT_IN } from './constants';
 import { unlock } from './lock-unlock';
 
 const {
 	DropdownMenuV2: DropdownMenu,
-	DropdownMenuCheckboxItemV2: DropdownMenuCheckboxItem,
+	DropdownMenuGroupV2: DropdownMenuGroup,
+	DropdownMenuItemV2: DropdownMenuItem,
+	DropdownMenuSeparatorV2: DropdownMenuSeparator,
+	DropdownSubMenuV2: DropdownSubMenu,
+	DropdownSubMenuTriggerV2: DropdownSubMenuTrigger,
 } = unlock( componentsPrivateApis );
 
+const FilterText = ( { activeElement, filterInView, filter } ) => {
+	if ( activeElement === undefined ) {
+		return filter.name;
+	}
+
+	if (
+		activeElement !== undefined &&
+		filterInView?.operator === OPERATOR_IN
+	) {
+		return sprintf(
+			/* translators: 1: Filter name. 2: Filter value. e.g.: "Author is Admin". */
+			__( '%1$s is %2$s' ),
```

Comment:
I wonder how [more synthetic languages](https://en.wikipedia.org/wiki/Inflection) are handled in this sort of UI (e.g. Notion). I'm not discouraging this bit of string interpolation, just pointing out that we should see how others have done this, so that later we are sure that this UI is localisable.

Reply:
At some point, the proposal was `Field: value`. Though it didn't scale well when you have more operators (in, not in, starts with, etc.).

This is how notion does it (suffers from the same issue):

https://github.com/WordPress/gutenberg/assets/583546/49045409-6d9f-4f30-a0d6-b8313b0aa37f

This is linear (same issue):

https://github.com/WordPress/gutenberg/assets/583546/95a3bdbe-6185-40ab-a24e-0d11992fcebe

The same in other platforms (github, etc.).

I find synthetic languages difficult for this use case because the individual parts (field name, value) are translated separately, and you have no context what they are when translating this string as a whole. Happy to look at examples to improve it.

Reply:
This is German:

https://github.com/WordPress/gutenberg/assets/583546/c9945f47-ef49-4b67-b15c-9b86e4c6a031

The gender does affect other parts of the sentence, though the general structure is the same – I'd think what we have here is fine.

Reply:
Thanks for having a look! Yeah, I think it's doable, and at worst there might be a handful of languages in which the UI reads slightly awkwardly but remains understandable.